paper_trail.gemspec

```diff
@@ -19,9 +19,7 @@ has been destroyed.
   s.email = "jared@jaredbeck.com"
   s.license = "MIT"
 
-  s.files = `git ls-files -z`.split("\x0").select { |f|
-    f.match(%r{^(Gemfile|LICENSE|lib/|paper_trail.gemspec)})
-  }
+  s.files = Dir.glob(["lib/**/*", "LICENSE"])
```

Comment:
```suggestion
  s.files = Dir.glob(["CHANGELOG.md", "doc/*", "lib/**/*", "LICENSE", "README.md"])
```

Reply:
Hm, there are development-specific doc/ files. Perhaps only include the one with the warning?